cmake/partition_manager.cmake

```diff
@@ -52,23 +83,12 @@ if(FIRST_BOILERPLATE_EXECUTION)
       ${PM_MCUBOOT_SECONDARY_ADDRESS}
       )
 
-    get_property(
```

Comment:
It is not clear to me that the functionality provided here is replaced with the change. Please double check.

Reply:
Added now